### PR TITLE
Adding comments to modules, updating dates to 2017

### PR DESCRIPTION
**Pull request by Phansa**
I have added some comments to the sample, lastseen and stickychan modules. I have updated the copyright dates on many files from 2016 to 2017 and added liscene information to some files that were missing it.

### CODE REVIEW

File: modules/lastseen.cpp [hunk 1]
```diff
@@ -128,6 +130,7 @@ class CLastSeenMod : public CModule {
         return false;
     }
 
+    // Provides GUI to configure this module by adding a widget to user page in webadmin.
```
Comment (DarthGandalf) [inline modules/lastseen.cpp]: No, it doesn't.
What would be such configuration?

Comment (Phansa) [inline modules/lastseen.cpp]: I wasn't sure about this one so I will comment it out. Should I seperate one commit of updating all the dates and one of adding comments to modules?

Comment (DarthGandalf) [inline modules/lastseen.cpp]: Such separation is fine.

Comment (Phansa) [inline modules/lastseen.cpp]: Should I fix the commit messages and create another pull request/close this one?

Comment (DarthGandalf) [inline modules/lastseen.cpp]: Just force-push to this one.

**Comment by DarthGandalf** [comment]
temp and temp2?..

**Comment by Phansa** [comment]
The temp/temp2 was just when I was fixing the commits. It should all be correct now with these last 3 commits.

File: modules/lastseen.cpp [hunk 2]
```diff
@@ -95,6 +96,7 @@ class CLastSeenMod : public CModule {
     bool WebRequiresAdmin() override { return true; }
     CString GetWebMenuTitle() override { return "Last Seen"; }
 
+    // Provides GUI to configure this module by adding a widget to user page in webadmin.
```
Comment (DarthGandalf) [inline modules/lastseen.cpp]: There is no configuration for this module.

**Pull request merged**